scio-core/src/main/scala/com/spotify/scio/ScioResult.scala

```diff
@@ -76,6 +77,16 @@ class ScioResult private[scio] (val internal: PipelineResult,
     this
   }
 
+  /** Wait until the pipeline finishes with the State Done (as opposed to Cancelled). Throws
```

Comment:
Nitpick: can you use JavaDoc style as everywhere else? You can set it to default for scala in IntelliJ.
Something like:
```
/**
 * Blah blah ...
 * blah.
 */```